### DISCUSSION TITLE
Query a markdown table

### DISCUSSION THREAD

**Discussion by reagle** [sytone/obsidian-queryallthethings#36] [Q&A] [answered]
If I had the following markdown table, could this plugin sum Jim's values, or sum Paul's values when greater than Jim's?

| Week  | Jim  | Paul  |
|:-----:|:----:|:----:|
|   3   |  85  |  92  |
|   4   |  75  | 115  |
|   5   |  0   |  78  |
|   6   |  53  |  56  |
|   7   |  85  |  75  |
|   8   | 101  |  58  |
|   9   | 111  |  93  |
|  10   |  74  |  88  |
|  11   | 101  |  91  |

**Comment by sytone** [comment]
yes, depends on how complicated you want to get. You could just reference the md file with the table and then show the sum of the values for both, then you just see which is bigger. Or you could use conditionals in the handlebars template to hide or show depending on which value is greater.

**Comment by sytone** [comment, marked as answer]
Assuming the table is in a file called QATTISSUE36.md and you had added it in settings to be loaded by the plugin this will show you who has the most values. I left all the details in there, but you can update and modify as needed. 

```qatt
query: |
  SELECT SUM(Jim) AS JimTotal, SUM(Paul) AS PaulTotal, SUM(Jim) > SUM(Paul) AS JimLarger
  FROM QATTISSUE36
template: |
  {{#each result}}
  {{JimTotal}} - {{PaulTotal}}
  {{JimLarger}}
  {{#if JimLarger}}Jim has {{JimTotal}}{{else}}Paul has {{PaulTotal}}{{/if}}
  {{/each}}
```
